### ISSUE TITLE
Accessibility improvements - Select component

### ISSUE THREAD

**Issue by robinmolen** [open-formulieren/open-forms#5050]
This ticket describes accessibility improvements for the SDK select component. It could be that we cannot fix the issues for this component, due to the `react-select` library component we use. Lets start with a time-box of 1/2 day, and assess if this is possible

- [ ] 4.1.2 (name, role, value) The select component contains problems with name and role on the select element
- [ ] 4.1.2 (name, role, value) The select component parent div contains aria-selected="true". This attribute isn't allowed on div elements
- [ ] 4.1.2 (name, role, value) The select component parent div misses the `aria-controls` attribute
